### ISSUE TITLE
统计中文版 API 中还未翻译的部分

### ISSUE THREAD

**Issue by robin978** [Oneflow-Inc/oneflow-api-cn#80]

**Comment by robin978** [comment]
# oneflow.tensor:
### 未翻译：
[oneflow.Tensor.addcmul](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addcmul) , [oneflow.Tensor.addcmul_](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addcmul_) , [oneflow.Tensor.addmm](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.addmm) , [oneflow.Tensor.amax](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.amax) ，[oneflow.Tensor.amin](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.amin) ,[oneflow.Tensor.asinh](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.asinh) , [oneflow.Tensor.byte](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.byte) , [oneflow.Tensor.cosh](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.cosh) , [oneflow.Tensor.device](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.device) , [oneflow.Tensor.global_to_global](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.global_to_global) , [oneflow.Tensor.grad](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.grad) ， [oneflow.Tensor.grad_fn](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.grad_fn) , [oneflow.Tensor.half](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.half) , [oneflow.Tensor.is_contiguous](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.is_contiguous) , [oneflow.Tensor.is_cuda](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.is_cuda) , [oneflow.Tensor.is_global](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.is_global) , [oneflow.Tensor.is_lazy](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.is_lazy) , [oneflow.Tensor.is_leaf)](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.is_leaf) , [oneflow.Tensor.local_to_global](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.local_to_global) , [oneflow.Tensor.log](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.log) , [oneflow.Tensor.new_empty](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.new_empty) , [oneflow.Tensor.new_zeros](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.new_zeros) , [oneflow.Tensor.placement](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.placement) , [oneflow.Tensor.register_hook](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.register_hook) , [oneflow.Tensor.requires_grad](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.requires_grad) , [oneflow.Tensor.requires_grad_](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.requires_grad_) , [oneflow.Tensor.retain_grad](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.retain_grad) , [oneflow.Tensor.sbp](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.sbp) , [oneflow.Tensor.sin_](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.sin_) , [oneflow.Tensor.sqrt](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.sqrt) , [oneflow.Tensor.square](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.square) , [oneflow.Tensor.storage_offset](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.storage_offset) , [oneflow.Tensor.swapdims](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.swapdims) , [oneflow.Tensor.to_consistent](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.to_consistent) , [oneflow.Tensor.unbind](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.unbind) , [oneflow.Tensor.view_as](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.view_as) , [oneflow.Tensor.zero_](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.zero_) , 

### 缺少docstring
[oneflow.Tensor.data](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.data) ,[oneflow.Tensor.detach](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.detach) , [oneflow.Tensor.dtype](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.dtype) ,  [oneflow.Tensor.norm](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.norm) , [oneflow.Tensor.shape](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.shape) , [oneflow.Tensor.stride](https://start.oneflow.org/oneflow-api-cn/tensor.html#oneflow.Tensor.stride) ,

**Comment by robin978** [comment]
# oneflow.nn
### 缺少docstring：
[oneflow.nn.COCOReader](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.COCOReader) , [oneflow.nn.CoinFlip](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.CoinFlip) ,[oneflow.nn.CropMirrorNormalize](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.CropMirrorNormalize) , [oneflow.nn.FusedMLP](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.FusedMLP) , [oneflow.nn.ModuleDict](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.ModuleDict) , [oneflow.nn.ModuleList](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.ModuleList) , [oneflow.nn.OFRecordImageDecoder](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.OFRecordImageDecoder) , [oneflow.nn.OFRecordImageDecoderRandomCrop](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.OFRecordImageDecoderRandomCrop) , [oneflow.nn.OFRecordRawDecoder](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.OFRecordRawDecoder) , [oneflow.nn.OFRecordReader](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.OFRecordReader) , [](

### 公式格式：
[oneflow.nn.FakeQuantization](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.FakeQuantization) , [oneflow.nn.LayerNorm](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.LayerNorm) , [](https://start.oneflow.org/oneflow-api-cn/nn.html#oneflow.nn.MovingAverageMinMaxObserver) , [](

**Comment by robin978** [comment]
# oneflow.nn.functional:
### 未翻译：
[oneflow.nn.functional.conv_transpose1d](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.conv_transpose1d) , [oneflow.nn.functional.conv_transpose2d](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.conv_transpose2d) , [oneflow.nn.functional.conv_transpose3d](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.conv_transpose3d) , [oneflow.nn.functional.cross_entropy](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.cross_entropy) , 

###  没有docstring：
[oneflow.nn.functional.hardshrink](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.hardshrink) , [oneflow.nn.functional.threshold](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.threshold) , [oneflow.nn.functional.softshrink](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.softshrink) , 

### 公式格式：
[](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.leaky_relu) , [oneflow.nn.functional.elu](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.elu) , [oneflow.nn.functional.selu](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.selu) , [oneflow.nn.functional.sigmoid](https://start.oneflow.org/oneflow-api-cn/functional.html#oneflow.nn.functional.sigmoid) ,

**Comment by robin978** [comment]
# oneflow.linalg
### 表格格式：
[oneflow.linalg.matrix_norm](https://start.oneflow.org/oneflow-api-cn/linalg.html#oneflow.linalg.matrix_norm) , [oneflow.linalg.norm](https://start.oneflow.org/oneflow-api-cn/linalg.html#oneflow.linalg.norm) , [oneflow.linalg.vector_norm](https://start.oneflow.org/oneflow-api-cn/linalg.html#oneflow.linalg.vector_norm) ,

**Comment by robin978** [comment]
# oneflow.init
### 未翻译：
[oneflow.nn.init.orthogonal_](https://start.oneflow.org/oneflow-api-cn/nn.init.html#oneflow.nn.init.orthogonal_)

**Comment by robin978** [comment]
# oneflow.optim
### 未翻译：
[oneflow.optim.Adam.support_sparse](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.Adam.support_sparse) , [oneflow.optim.Optimizer.step](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.Optimizer.step) , [oneflow.optim.SGD.step](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.SGD.step) , [oneflow.optim.lr_scheduler.CosineAnnealingLR.get_lr](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.lr_scheduler.CosineAnnealingLR.get_lr) , ,[oneflow.optim.lr_scheduler.LambdaLR.step](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.lr_scheduler.LambdaLR.step) , [oneflow.optim.lr_scheduler.ReduceLROnPlateau](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.lr_scheduler.ReduceLROnPlateau)
每个类函数下的get_lr 函数未翻译，在翻译的时候无法找到该函数，无法生成页面，

### 公式格式：
[oneflow.optim.RMSprop](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.RMSprop) , [oneflow.optim.SGD](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.SGD) , [oneflow.optim.lr_scheduler.CosineDecayLR](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.lr_scheduler.CosineDecayLR) , [oneflow.optim.lr_scheduler.PolynomialLR](https://start.oneflow.org/oneflow-api-cn/optim.html#oneflow.optim.lr_scheduler.PolynomialLR) , [](

**Comment by robin978** [comment]
# oneflow.nn,Module:
很多子函数都没有 docstring [oneflow.nn.Module](https://start.oneflow.org/oneflow-api-cn/module.html#oneflow.nn.Module)

**Comment by robin978** [comment]
# oneflow.nn.Graph:
### 未翻译：
[oneflow.nn.graph.graph_config.GraphConfig](https://start.oneflow.org/oneflow-api-cn/graph.html#oneflow.nn.graph.graph_config.GraphConfig) , [oneflow.nn.graph.block_config.BlockConfig](https://start.oneflow.org/oneflow-api-cn/graph.html#oneflow.nn.graph.block_config.BlockConfig)

**Comment by robin978** [comment]
# oneflow.nn.image:
均未翻译 https://start.oneflow.org/oneflow-api-cn/image.html

**Comment by robin978** [comment]
# oneflow.utils:
### 未翻译：
[oneflow.utils.from_torch](https://start.oneflow.org/oneflow-api-cn/utils.html#oneflow.utils.from_torch) , [oneflow.utils.to_torch](https://start.oneflow.org/oneflow-api-cn/utils.html#oneflow.utils.to_torch)

### 缺少docstring：
[oneflow.utils.data.ConcatDataset](https://start.oneflow.org/oneflow-api-cn/utils.html#oneflow.utils.data.ConcatDataset) , [oneflow.utils.data.DataLoader](https://start.oneflow.org/oneflow-api-cn/utils.html#oneflow.utils.data.DataLoader) , 子函数缺少docstring , [oneflow.utils.data.IterableDataset](https://start.oneflow.org/oneflow-api-cn/utils.html#oneflow.utils.data.IterableDataset) , [](oneflow.utils.data.IterableDataset

**Comment by KSnow616** [comment]
我来负责下nn.functional 到 oneflow.utils 这部分吧

**Comment by doombeaker** [comment]
我在第一个 comment 中做了分工。大家没问题的话，就可以开始了。
